monai/visualize/img2tensorboard.py

```diff
@@ -192,7 +192,7 @@ def plot_2d_or_3d_image(
         dataformats = "HW"
         for j, d2 in enumerate(d[:max_channels]):
             d2 = rescale_array(d2, 0, 1)
-            writer.add_image(f"{tag}_{dataformats}_{j}", d2, step, dataformats=dataformats)
+            writer.add_image(f"{tag}_{dataformats}/{j}", d2, step, dataformats=dataformats)
```

Comment:
In order to preserve existing behaviour for users, this change should be controlled by a flag passed in as an argument to `plot_2d_or_3d_image`. with the default being the existing behaviour.

Reply:
The current behavior is inconsistent between different image types. Currently, multiple 3D images are plotted in the same tag while multiple 2D images are plotted in different tags. I would think consistency would be preferred over preserving the current behavior? Also, the flag should probably control all image sizes and not just 2D?

Reply:
Hi @MattTheCuber, we can simply add a flag for this multi-channel 2d image plt. Thanks.

Reply:
I'm kind of confused what behavior you desire. Should I make a flag for controlling all plotting (2d & 3d)? Should I make flags for 2D and 3D with 2D default on and 3D default off? Should I simply add a flag for the change I made?

Reply:
I agree, refactoring this would be nice. As I said previously, the current behavior is quite disjoint.